client/layout/sidebar-v2/navigator/navigator-menu-item.tsx

```diff
@@ -41,7 +41,9 @@ export const SidebarNavigatorMenuItem = ( {
 					'is-active': isSelected,
 				} ) }
 				onClick={ () => onClickMenuItem( link ) }
+				href={ link }
```

Comment:
We also do a `page( path );` on click. I think we can remove that now.

Reply:
I've been looking into this today. I agree we should be able to remove calls to `page`, but without it, for some reason, it seems like our check for `isMenuItemSelected` executes _before_ the page updates. This means the highlighted menu item becomes the _previously_ viewed page instead of the current one. So frustrating! But I'm investigating.

Reply:
I found a solution, cribbing from the approach @jkguidaven pasted from the old single-site sidebar menu: adding `context.path` to the sidebar props allows us to react to it as it changes.